src/bin/cargo/commands/config.rs

```diff
@@ -4,7 +4,6 @@ use cargo::ops::cargo_config;
 pub fn cli() -> Command {
     subcommand("config")
         .about("Inspect configuration values")
-        .after_help("Run `cargo help config` for more detailed information.\n")
```

Comment:
Could we split this out into its own commit so we can just revert this when needed?

Reply:
Split! See https://github.com/rust-lang/cargo/pull/11473/commits/53fd81552299d0c0a04bc5bfe682c868262d15b0